### PR TITLE
caddy: add page

### PR DESCRIPTION
**Pull request by ddgond**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/caddy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# caddy
+
+> A powerful, enterprise-ready, open source web server with automatic HTTPS written in Go.
```
Comment (bl-ue) [inline pages/common/caddy.md:3]:
```suggestion
> A powerful, enterprise-ready, open source web server with automatic HTTPS, written in Go.
```

File: pages/common/caddy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://caddyserver.com>.
+
+- Start Caddy in the foreground:
+
+`caddy run`
+
+- Start Caddy with a specified Caddyfile:
```
Comment (bl-ue) [inline pages/common/caddy.md:10]:
```suggestion
- Start Caddy with the specified Caddyfile:
```

File: pages/common/caddy.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Start Caddy in the background:
+
+`caddy start`
+
+- Stop a background Caddy process:
+
+`caddy stop`
+
+- Run a simple file server on a specified port with a browsable interface:
```
Comment (bl-ue) [inline pages/common/caddy.md:22]:
```suggestion
- Run a simple file server on the specified port with a browsable interface:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
A few grammar suggestions. :)

**Comment by ddgond** [comment]
Thanks for the suggestions! I've also added the trailing newline that was originally missing.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/caddy.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+> A powerful, enterprise-ready, open source web server with automatic HTTPS, written in Go.
+> More information: <https://caddyserver.com>.
+
+- Start Caddy in the foreground:
+
+`caddy run`
+
+- Start Caddy with the specified Caddyfile:
+
+`caddy run --config /example/Caddyfile`
```
Comment (marchersimon) [inline pages/common/caddy.md:12]:
```suggestion
`caddy run --config {{path/to/Caddyfile}}`
```
We use this syntax for specifying files.

File: pages/common/caddy.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`caddy start`
+
+- Stop a background Caddy process:
+
+`caddy stop`
+
+- Run a simple file server on the specified port with a browsable interface:
+
+`caddy file-server --listen :8000 --browse`
```
Comment (marchersimon) [inline pages/common/caddy.md:24]:
```suggestion
`caddy file-server --listen :{{8000}} --browse`
```
We wrap variable options in `{{ }}`, so clients can highlight them.

File: pages/common/caddy.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run a reverse proxy server:
+
+`caddy reverse-proxy --from :80 --to localhost:8000`
```
Comment (marchersimon) [inline pages/common/caddy.md:28]:
```suggestion
`caddy reverse-proxy --from :{{80}} --to localhost:{{8000}}`
```

**Review by marchersimon — CHANGES_REQUESTED** [review]
Just a few syntax related suggestions

**Review by bl-ue — APPROVED** [review]
Looks good modulo @marchersimon's suggestions :)

**Comment by ddgond** [comment]
Thanks for the quick responses! Should be all set now.

**Review by marchersimon — APPROVED** [review]
Thanks @ddgond

**Comment by bl-ue** [comment]
>Thanks for the quick responses! Should be all set now.

I'm sorry, I should've been faster — I didn't realize that 3 hours ago I was [requested](https://github.com/tldr-pages/tldr/pull/6115#event-4879417085) for review 😂

**Review by mfrw — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by bl-ue** [comment]
Welcome to tldr-pages, @ddgond! :)

**Pull request merged**